frontend/src/routes/Credentials/CredentialsForm.tsx

```diff
@@ -477,8 +486,8 @@ export function CredentialsForm(
       { path: 'Secret[0].stringData.noProxy', setState: setNoProxy },
       { path: 'Secret[0].stringData.bucket', setState: setBucketName },
       { path: 'Secret[0].stringData.region', setState: setAwsS3Region },
-      { path: 'Secret[0].stringData.aws_access_key_id', setState: setAwsAccessKeyID },
-      { path: 'Secret[0].stringData.aws_secret_access_key', setState: setAwsSecretAccessKeyID },
+      { getter: getValue.bind(null, 'aws_access_key_id'), setState: setAwsAccessKeyID },
```

Comment:
Just wanna make sure this change will not mess the AWS credential up. I'll test this PR locally and let you know how it goes.

Reply:
Worked amazing @jeswanke !